File: blog-sever/server-java/src/main/java/com/site/blog/model/dto/IndexDto.java
package com.site.blog.model.dto;

import com.site.blog.model.entity.BlogConfig;
import lombok.Data;

import java.util.List;


@Data
public class IndexDto {
    private int categoryCount;
    private int blogCount;
    private int linkCount;
    private int tagCount;
    private int commentCount;
    private List<BlogConfig> sysList;
}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/model/dto/IndexDto.kt
package com.site.blog.model.dto

import com.site.blog.model.entity.BlogConfig

data class IndexDto(
    var categoryCount: Int? = 0,
    var blogCount: Int? = 0,
    var linkCount: Int? = 0,
    var tagCount: Int? = 0,
    var commentCount: Int? = 0,
    var sysList: List<BlogConfig>? = null
)